packages/buendia-site-reference/data/usr/share/buendia/site/40-site

```diff
@@ -0,0 +1,33 @@
+# Site configuration for integration testing on an Edison.
+
+SERVER_OPENMRS_USER=buendia
+SERVER_OPENMRS_PASSWORD=buendia
+
+MONITORING_LIMITS='
+100000 /var/log/large/requests
+200000 /var/log
+
+300000 /var/cache
+
+50000 /var/backups/buendia/packages
+50000 /var/backups/buendia/backup*
+100000 /var/backups/buendia
+
+100000 /usr/share/buendia/packages
+'
+
+# Default network interfaces on the Intel NUC
+NETWORKING_WIFI_INTERFACE=wlp2s0
+NETWORKING_ETHERNET_INTERFACE=enp3s0
+
+# Join the existing "buendia" network as provided by the Nanostation
+NETWORKING_SSID=buendia
+NETWORKING_PASSWORD=buendia
```

Comment:
The password can't be "buendia", can it?  I thought we used "plumpynut" because it had to be at least 8 characters.

Reply:
yeahhhh